src/components/functional/GlobalBrush.jsx

```diff
@@ -50,7 +52,7 @@ class GlobalBrush extends React.Component {
                         height = {this.props.height}
                         margin = {BRUSH_MARGIN}
                         xScale = {d3.time.scale().domain([startDate, endDate]).range([0, this.props.containerWidth - BRUSH_HORIZONTAL_MARGIN])}
-                        extent = {[startDate, endDate]}
+                        extent = {[this.state.extent[0], this.state.extent[1]]}
```

Comment:
Not sure I get this `extent` variable. Since it's only ever `[startDate, endDate]` you can use those directly **or** at least shorten this to `extent={this.state.extent}`.